src/lib/statistics/statistics_objects/min_max_filter.cpp

```diff
@@ -158,6 +158,49 @@ bool MinMaxFilter<T>::does_not_contain(const PredicateCondition predicate_condit
       const auto value2 = boost::get<T>(*variant_value2);
       return value >= max || value2 <= min;
     }
+    case PredicateCondition::NotLike:
```

Comment:
I am surprised to see that both predicate conditions are handled with the same block? Are you sure?

Reply:
In any case, NotLike does not seem to be tested.

Reply:
Actually, now that you are mentioning it I'm not so sure anymore. I will write some tests and rework the code and make sure that everything is working.